File: app/src/androidTest/java/com/leaguechampions/robots/ChampionDetailsRobot.java
package com.leaguechampions.robots;

import com.leaguechampions.R;

import static android.support.test.espresso.Espresso.onView;
import static android.support.test.espresso.assertion.ViewAssertions.matches;
import static android.support.test.espresso.matcher.ViewMatchers.withId;
import static android.support.test.espresso.matcher.ViewMatchers.withText;

public class ChampionDetailsRobot {

    public ChampionDetailsRobot verifyChampionIsDisplayed(String championId) {
        onView(withId(R.id.activity_champion_details_tvName))
                .check(matches(withText(championId)));
        return this;
    }
}

File: app/src/androidTest/java/com/leaguechampions/ui/championdetails/ChampionDetailsRobot.kt
package com.leaguechampions.ui.championdetails

import androidx.test.espresso.Espresso.onView
import androidx.test.espresso.assertion.ViewAssertions.matches
import androidx.test.espresso.matcher.ViewMatchers.withId
import androidx.test.espresso.matcher.ViewMatchers.withText
import com.leaguechampions.R

class ChampionDetailsRobot {

    fun verifyChampionIsDisplayed(championId: String): ChampionDetailsRobot {
        onView(withId(R.id.activity_champion_details_tvName))
                .check(matches(withText(championId)))
        return this
    }
}
